RCSwitch.cpp

```diff
@@ -466,10 +465,12 @@ void RCSwitch::transmit(int nHighPulses, int nLowPulses) {
   }
 #endif
 
-  digitalWrite(this->nTransmitterPin, HIGH);
-  delayMicroseconds( this->protocol.pulseLength * nHighPulses);
-  digitalWrite(this->nTransmitterPin, LOW);
-  delayMicroseconds( this->protocol.pulseLength * nLowPulses);
+  for(uint8_t i = 0; i < COUNT_OF(sequence.pulses); i++)
+  {
```

Comment:
I think it's a good idea to match the predominante code formatting style (i.e. braces on the same line as `for`, and a space after `for`). I know, everybody has different preferences, but it's a good idea to at least stay consistent with the surrounding code, don't you agree?